src/medCore/process/medAbstractProcess.h

```diff
@@ -32,6 +32,8 @@ class MEDCORE_EXPORT medAbstractProcess : public dtkAbstractProcess
     medAbstractProcess( medAbstractProcess * parent = NULL );
     virtual ~medAbstractProcess();
 
+    using dtkAbstractProcess::setInput;
```

Comment:
medAbstractWhateverKindOfProcess do not belong to another plugins, it's defined in the process layer which comes from the libs ships by medInria (medCore, medProcess, medGui etc..)

it could be for instance medAbstractSubtractImageProcess (see the dev branch in the medInria-public repo) which expose: 
```cpp
void setInput1(medAbstractImageData *data);
void setInput2(medAbstractImageData *data);
medAbstractImageData * output() const;
```